app/uk/gov/hmrc/cipphonenumberverification/services/NotificationService.scala

```diff
@@ -51,18 +51,18 @@ class NotificationsService @Inject()(govUkConnector: GovUkConnector)
     def failure(err: UpstreamErrorResponse) = {
       err.statusCode match {
         case NOT_FOUND =>
+          logger.warn("Notification ID not found")
           val json = Json.parse(
```

Comment:
could we change this to Json.toJson write converters

Reply:
sure, will do that now